README.md

```diff
@@ -11,6 +11,8 @@ _Note: Stencil requires the Node.js runtime environment,
 version 10.x and 12.x (Recommended) are supported.
 We do not yet have support for versions greater than Node 12._
 
+_Review package.json for the required npm version, <7.0.0 at the time of writing_
```

Comment:
Do you mind update your PR also with the lowest npm boundary? 
Currently we support node 10.17 and it comes with `npm 6.11.3`

Reply:
@jairo-bc thats been done now